tests/ArrayParserTest.php

```diff
@@ -25,9 +25,10 @@ public function testParser(): void
         $this->assertSame([0 => '1', 1 => '2', 2 => '3'], $arrayParse->parse('{1,2,3}'));
         $this->assertSame([0 => '1', 1 => '-2', 2 => null, 3 => '42'], $arrayParse->parse('{1,-2,NULL,42}'));
         $this->assertSame([[0 => 'text'], [0 => null], [0 => '1']], $arrayParse->parse('{{text},{NULL},{1}}'));
+        $this->assertSame([0 => ''], $arrayParse->parse('{""}'));
         $this->assertSame(
-            [[0 => '[",","null",true,"false","f"]'], 1 => ''],
-            $arrayParse->parse('"{"[\",\",\"null\",true,\"false\",\"f\"]"}"')
+            [[0 => '[",","null",true,"false","f"]']],
```

Comment:
add parse test for complex string: $arrayParse->parse('"{"[\",\",\"null\",true,\"false\",\"f\"]"}"')

Reply:
This is incorrect test. PostgreSQL will never return string `'"{"[",","null",true,"false","f"]"}"'` for `array` value. It shold be starts with `{` and ends with `}`. In the example it starts and ends with `"`.

Reply:
If i save to column 'varchararray_col' this data: new ArrayExpression([0 => '[",","null",true,"false","f"]'])
then after read i have this: string(41) "{"[\",\",\"null\",true,\"false\",\"f\"]"}"
Please add test for parse this string

Reply:
https://github.com/yiisoft/db-pgsql/tree/arr_test

Reply:
Ok, added as additional array value.